demo/floating-buttons.html

```diff
@@ -1,13 +1,13 @@
 <html>
 <head>
 	<title>d2l-floating-buttons</title>
-	<script src="https://s.brightspace.com/lib/webcomponentsjs/0.7.21/webcomponents.min.js"></script>
+	<script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 	<link rel="import" href="../../d2l-demo-template/d2l-demo-template.html">
 	<link rel="import" href="../d2l-button.html">
 	<link rel="import" href="../d2l-floating-buttons.html">
 
 	<style>
-		.d2l-demo-actions > button {
+		.d2l-demo-actions > d2l-button > button {
```

Comment:
Kind of sucks that the shadow DOM is leaking here -- could the margin not be applied to `d2l-button` directly and still work? I think it should work...